src/hiveboard_bridge/src/main.cpp

```diff
@@ -47,7 +44,12 @@ int main(int argc, char** argv) {
         // Publish on moveby
         moveByPublisher.publish(moveByMessage);
 
-        // TODO Send ack/response over TCP
+        // Send ack/response
+        FunctionCallResponseDTO functionCallResponse(GenericResponseStatusDTO::Ok, "moveBy OK");
```

Comment:
The details are more for errors to log on the other side, I thinks it's fine to put an empty string on ok